ompi/request/request.c

```diff
@@ -50,6 +53,10 @@ ompi_request_fns_t               ompi_request_functions = {
     ompi_request_default_wait_all,
     ompi_request_default_wait_some
 };
+#if MPI_VERSION >= 4
+ompi_request_t ompi_request_empty_send = {{{{0}}}};
+extern int ompi_pml_base_deprecation_warn_level;
```

Comment:
`extern` is not right here, unless `ompi_pml_base_deprecation_warn_level` is defined in another TU but then it should go into a header.